### ISSUE TITLE
[KEYBOARD]: Map - Focus is moved to the search results when users press arrow keys or +/-

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#490]
## Issue
Focus is being moved from the map to the search results when keyboard users press an arrow key to shift the map in a direction, or when users press plus or minus to zoom the map. This was noted as an SC 3.3.2 violation. Animated GIF attached below.

## Related Issues
* https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/491
* https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/515

## Audit Finding
* Note 5, Defect 1 of 2

## Acceptance Criteria
* As a keyboard user, I would like the map to retain focus when I press an arrow key, or any other keyboard shortcut. The number of results should update as before, but the yellow focus halo should stay on the map.

## Environment
* MacOS Mojave
* Chrome latest
* https://staging.va.gov/find-locations/

## WCAG or Vendor Guidance (optional)
* [On Input: Understanding SC 3.2.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html)

## Screenshots or Trace Logs
![map-focus-change-smaller.gif](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/5c21b5e3-c1dd-4f04-8b41-f4563c5ece76)

**Comment by Karak888** [comment]
@Karak888 will discuss with @jason-gcio to understand if this is being duplicated?

**Comment by Karak888** [comment]
crossteam meeting will be good to have with the new Facility Locator teeam as they come up to speed.

**Comment by kelsonic** [comment]
I just want to reaffirm that this is what we want:

**Current behavior** is that when a user is focused on the map and presses either `+` or `-`, it zooms the user in/out and when the new results come in we focus on the results.

**Expected behavior** is that when a user is focused on the map and presses either `+` or `-`, it zooms the user in/out and when the new results come in focus does not change.

Just double-checking that we hypothesize that the expected behavior is better than the current behavior for our end users UX-wise. @Karak888 @1Copenut

**Comment by 1Copenut** [comment]
@kelsonic Great question. I wrote the ticket based on the auditor's assessment that moving focus was a violation of WCAG Success Criteria 3.2.2. We are not informing users in text or instructions that a change in the map will cause focus to be moved off the map "spot" so to speak, and that triggered the notation.

When I was testing, the behavior felt disruptive to move back to the list of facilities. If a keyboard user has made the effort to `TAB` through to the map (past all of the tab stops in the list) they have self-identified as wanting to search by map visual orientation. This feels like a stronger solution for all users. Mouse users will still have full access to the click events on map or list, keyboard and screen reader users will not be pulled out of the map back to the list.

**Comment by 1Copenut** [comment]
Closing this as not a defect per guidance from VA 508 office.

Please see my PR comment: https://github.com/department-of-veterans-affairs/vets-website/pull/10819#issuecomment-538563246

/cc @Karak888
